pegasus/sites.v3/code.org/public/teacher-dashboard/sections.haml

```diff
@@ -157,3 +158,33 @@ content-type: text/ng-template
 %br
   %p= I18n.t('dashboard_privacy_document_link')
 %br/
+
+%script{src: 'https://apis.google.com/js/api.js'}
+#roster-dialog
+
+:scss
+  .modal-backdrop {
+    position: fixed;
+    bottom: 0;
+    opacity: 0.8;
+  }
+  .modal {
+    position: fixed;
+    display: block;
+    top: 10%;
+    left: 50%;
+    bottom: auto;
+    right: auto;
+    z-index: 1050;
+    background-color: #fff;
+    border-radius: 6px;
+    div {
+      text-align: left;
+    }
+    button {
+      height: 38px;
+    }
+  }
+  #selectLanguage {
+    display: none;
+  }
```

Comment:
The problem here is that all our Dialog components depend on external CSS from Bootstrap.  Dashboard and Pegasus have different Bootstrap versions, so the styles don't match up 😞.

What's the best way to resolve the external CSS dependencies in the dialog?  Ideally we'd completely re-write the `BaseDialog` component to not depend on any external styles, but that's a big change I want to balance that against shipping soon.